### PR TITLE
Fixes to duplicate search query parameters and enable the search icon…

### PR DESCRIPTION
**Pull request by ironfroggy**
… as a button

### CODE REVIEW

**Comment by dpoirier** [comment]
One comment then :shipit:

**Comment by ironfroggy** [comment]
@dpoirier I don't see any comment.

File: opendebates/static/js/base/helpers.js [hunk 1]
```diff
@@ -232,4 +232,13 @@
   $('#add_question').on('keyup', 'textarea', updateTextLimitCounts);
   $('#add_question textarea').each(updateTextLimitCounts);
 
+  $('#search-full').one('keyup click', function() {
+    console.log('use full');
```
Comment (dpoirier) [inline opendebates/static/js/base/helpers.js]: We should remove these `console.log`s.

**Comment by dpoirier** [comment]
Sorry, forgot to hit the Comment button.

**Pull request merged**